cart.js

```diff
@@ -0,0 +1,120 @@
+let products=[{  
+    id:1,
+    name: "Perfume",
+    price: "$100",
+    category:"Cosmetics",
+    imageURL:"images/perfume.jpg" ,
+        },{
+    id:2,
+    name: "Lipstick",
+    price: "$25",
+    category:"Cosmetics",
+    imageURL:"images/lipstick.jpg" ,
+        },{
+    id:3,
+    name: "Apple Airpods",
+    price: "$100",
+    category:"Technology",
+    imageURL:"images/airpods.jpg" ,
+        }];
+
+let cards = document.createElement("div");
+let cardsContainer = document.getElementsByClassName("cards-container")[0];
+cardsContainer.appendChild(cards);
+
+products.map(function(product){
+    let card= document.createElement("div");
+    let cardImage= document.createElement("img");    
+    let firstRowContainer = document.createElement("div");
+    let secondRowContainer = document.createElement("div");
+    let quantityContainer = document.createElement("div");
+    let cardName= document.createElement("h3");
+    let cardPrice = document.createElement("span");
+    let cardCategory = document.createElement("span");
+    let deleteBtn = document.createElement("button");
+    let quantity = document.createElement("input");
+    let quantityText = document.createElement("span");
+    let totalPrice= document.getElementsByClassName("total-price")[0];
+
+    cardImage.src=product.imageURL;
+    cardName.innerHTML=product.name;
+    cardPrice.innerHTML=product.price;
+    cardCategory.innerHTML=product.category;
+    deleteBtn.innerHTML='<i class="fas fa-trash-alt"></i>';
+    quantityText.innerHTML="Quantity:";
+    quantity.type="number";
+    quantity.min="1";
+    quantity.placeholder="1";
+
+    card.appendChild(cardImage);
+    firstRowContainer.appendChild(cardName);
+    firstRowContainer.appendChild(cardPrice);
+    secondRowContainer.appendChild(cardCategory);
+    secondRowContainer.appendChild(deleteBtn);
+    quantityContainer.appendChild(quantityText);
+    quantityContainer.appendChild(quantity);
+    card.appendChild(firstRowContainer);
+    card.appendChild(secondRowContainer);
+    card.appendChild(quantityContainer);
+
+    let cardContainer = document.getElementsByClassName("card-container")[0];
+    cardContainer.appendChild(card);
+    
+    card.style.display="flex";
+    card.style.flexDirection="column";
+    card.style.border="solid 1px #3C9B8A";
+    card.style.padding="25px";
+    card.style.margin= "2%";
+    card.style.boxShadow="-5px 6px 16px -6px rgba(0,0,0,0.59)";
```

Comment:
You could move all of these styles into your css file by setting a class on card:
`card.className = "card";`
and then in style.css:
```
.card {
  display: flex;
  flex-direction: column;
  border: 1px solid #3C9B8A;
  padding: 25px;
  margin: 2%;
  box-shadow: -5px 6px 16px -6px rgba(0,0,0,0.59);
}
```

And you could do the same for all of the other styles below this line too.  I think that would be a little better for code organization - then all of the styles are in one place and easy to find, and the javascript file is smaller and easier to read.

Reply:
I was trying to find a way to move the style from the js file, thank you.

Reply:
👍 you’re welcome